tensorboard/webapp/metrics/views/main_view/filter_input_container.ts

```diff
@@ -84,9 +85,7 @@ export class MetricsFilterInputContainer {
       ),
       filter(([, tagFilterRegex]) => tagFilterRegex !== null),
       map(([tags, tagFilterRegex]) => {
-        return tags
-          .filter((tag: string) => tagFilterRegex!.test(tag))
-          .sort(compareTagNames);
+        return tags.filter((tag: string) => tagFilterRegex!.test(tag));
```

Comment:
I can tell that "sorting `compareTagNames` earlier" part is in `filtered_view_container.ts` 
does it also the same case in this `filter_input_container.ts`? what are the relationship between these two components? 
I searched where this `metrics-tag-filter` is applied but not see anywhere is using this 
https://cs.opensource.google/search?q=%22%27metrics-tag-filter%22&sq=&ss=tensorflow%2Ftensorboard

Reply:
Do not know why it is not picking up but it is used here: https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/metrics/views/main_view/main_view_component.ng.html;l=18

Two containers are related but are orthogonal. Input one renders autocomplete based on tag matches while the View one renders cards based on matches. They technically can share a common container parent that passes down list of tag matches but, currently, they are implemented as two separate containers with their own Observable streams both requiring similar optimizations.